tests/MongoDB.Driver.Core.Tests/Core/Clusters/DnsMonitorTests.cs

```diff
@@ -0,0 +1,327 @@
+﻿/* Copyright 2019-present MongoDB Inc.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+* http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Threading;
+using DnsClient;
+using DnsClient.Protocol;
+using FluentAssertions;
+using MongoDB.Bson.TestHelpers;
+using MongoDB.Driver.Core.Clusters;
+using MongoDB.Driver.Core.Misc;
+using MongoDB.Driver.Core.Servers;
+using Moq;
+using Xunit;
+
+namespace MongoDB.Driver.Core.Clusters
+{
+    public class DnsMonitorTests
```

Comment:
Given the number of classes changed, I'm a bit surprised that no new unit tests were required.  What do you think?